src/components/app-top-bar.js

```diff
@@ -382,8 +400,21 @@ const AppTopBar = ({ user, tabIndex, onChangeTab, userManager }) => {
                                 return <Tab key={tabName} label={label} />;
                             })}
                         </Tabs>
-                    )}
-                    {studyUuid && (
+                        <Box sx={styles.searchButton}>
+                            <Tooltip
```

Comment:
This tooltip was not here before.
Maybe you should ask a PO to confirm ?

Reply:
I confirmed with a PO, it's OK.